reactapp/src/api/api-routes.tsx

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
```

Comment:
@ralphsz28 these functions should be made to return values and save cookies, navigation code and error messages on screen should be handled in the page files
Regardless, I will approve this PR this time